gui-js/apps/minsky-electron/src/app/managers/ContextMenuManager.ts

```diff
@@ -348,18 +340,18 @@ export class ContextMenuManager {
 
   private static buildAndDisplayContextMenu(
     menuItems: MenuItem[],
-    mainWindow: Electron.BrowserWindow
+    mainWindow: Electron.BrowserWindow,
+    withHelp = true
```

Comment:
OK - another boolean flag I'm not totally happy with, but we can leave that for another refactor (if it happens).